modules/virt-understanding-live-migration.adoc

```diff
@@ -1,20 +1,19 @@
 // Module included in the following assemblies:
 //
 // * virt/live_migration/virt-live-migration.adoc
+// * virt/upgrading-virt.adoc
 
 [id="virt-understanding-live-migration_{context}"]
 = Understanding live migration
 
-Live migration is the process of moving a running virtual machine instance (VMI) to
-another node in the cluster without interrupting the virtual workload or access.
-If a VMI uses the `LiveMigrate` eviction strategy, it automatically migrates
-when the node that the VMI runs on is placed into maintenance mode. You can also
-manually start live migration by selecting a VMI to migrate.
+Live migration is the process of moving a running virtual machine instance (VMI) to another node in the cluster without interrupting the virtual workload or access. If a VMI uses the `LiveMigrate` eviction strategy, it automatically migrates when the node that the VMI runs on is placed into maintenance mode. You can also manually start live migration by selecting a VMI to migrate.
 
-Virtual machines must have a persistent volume claim (PVC) with a shared
-ReadWriteMany (RWX) access mode to be live migrated.
+You can use live migration if the following conditions are met:
 
-[NOTE]
-====
-Live migration is supported for virtual machines that are attached to an SR-IOV network interface only if the `sriovLiveMigration` feature gate is enabled in the HyperConverged Cluster custom resource (CR). When the `spec.featureGates.sriovLiveMigration` field is set to `true`, the `virt-launcher` pod runs with the `SYS_RESOURCE` capability. This might degrade its security.
-====
+* Virtual machines must have a persistent volume claim (PVC) with a shared `ReadWriteMany` (RWX) access mode to be live migrated.
+
+* The pod network binding must not be of the bridge interface type `()`.
+
+* Ports `49152` and `49153` must be available in the `virt-launcher` pod. If these ports are specified in the `masquerade` interface, live migration does not function.
```

Comment:
fyi, this limitation goes away in 4.10 due to this pr, https://github.com/kubevirt/kubevirt/pull/6323

Reply:
Thanks for letting me know. I'll make a note of this so that we can take it out.

Reply:
To track this, I opened CNV-14273 under our tech debt epic for 4.10.